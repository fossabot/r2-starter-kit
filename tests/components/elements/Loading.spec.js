import React from 'react';
import { shallow } from 'enzyme';
import Component from 'components/elements/Loading';

describe('<Loading />', () => {
  let wrapper;
  beforeAll(() => {
    wrapper = shallow(<Component />);
  });

  it('should render initial component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
