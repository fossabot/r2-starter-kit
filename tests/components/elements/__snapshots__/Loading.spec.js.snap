// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should render initial component 1`] = `
<div
  className="ts active inverted dimmer"
>
  <div
    className="ts text loader"
  >
    Loading...
  </div>
</div>
`;
